ql/src/java/org/apache/hadoop/hive/ql/exec/HashTableDummyOperator.java

```diff
@@ -45,8 +44,8 @@ protected void initializeOp(Configuration hconf) throws HiveException {
     super.initializeOp(hconf);
     TableDesc tbl = this.getConf().getTbl();
     try {
-      Deserializer serde = tbl.getDeserializerClass().newInstance();
-      SerDeUtils.initializeSerDe(serde, hconf, tbl.getProperties(), null);
+      AbstractSerDe serde = tbl.getSerDeClass().newInstance();
```

Comment:
Nit: please call this serDe for consistency.

Reply:
Done.